src/main/kotlin/no/nav/familie/ef/sak/blankett/BlankettService.kt

```diff
@@ -1,9 +1,6 @@
 package no.nav.familie.ef.sak.blankett
 
-import no.nav.familie.ef.sak.api.beregning.VedtakDto
-import no.nav.familie.ef.sak.api.beregning.VedtakService
-import no.nav.familie.ef.sak.api.beregning.fraDomene
-import no.nav.familie.ef.sak.api.beregning.tilInntekt
+import no.nav.familie.ef.sak.api.beregning.*
```

Comment:
Tror du behøver køra en pull på https://github.com/navikt/familie-settings før att få in de sista ændringarna

Reply:
oki takk :)